lms/djangoapps/course_api/tests/test_views.py

```diff
@@ -85,8 +85,13 @@ def test_as_inactive_user(self):
         self.verify_response(params={'username': inactive_user.username})
 
     def test_missing_username(self):
+        response_to_anonymous = self.verify_response(expected_status_code=200)
```

Comment:
Sounds good.  I don't know whether the latter is feasible, given our current access controls.  But it's worth digging into to check.